sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/EventPosition.cs

```diff
@@ -96,14 +96,7 @@ public struct EventPosition : IEquatable<EventPosition>
         /// <returns>The specified position of an event in the partition.</returns>
         ///
         public static EventPosition FromSequenceNumber(long sequenceNumber,
-                                                       bool isInclusive = true)
-        {
-            return new EventPosition
-            {
-                SequenceNumber = sequenceNumber,
```

Comment:
Why not just `ToString()` here?    It isn't possible for a long-to-string to be null or whitespace.

**Edit:** I see that we did that for offset too and, uh, I disagree with past Jesse and we should ridicule him.    Can we please streamline both and remove the private member?

Reply:
Updated!